packages/devtools/pubspec.yaml

```diff
@@ -18,7 +18,7 @@ environment:
   sdk: '>=2.3.0 <3.0.0'
 
 dependencies:
-  devtools_server: ^0.1.15
+  devtools_server: ^0.1.16
```

Comment:
0.1.16 does not exist - I don't know if this matters for pub. 0.2.2 is the latest released version of the server